api-reference/10 UI Widgets/dxMap/1 Configuration/apiKey/apiKey.md

```diff
@@ -5,7 +5,119 @@ default: { bing: '', google: '', googleStatic: '' }
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A key to authenticate the application within a map provider.
```

Comment:
- The application or the widget?
- What the String type is for?

Reply:
1. The widget (component), since there can be several Map instances. I'll fix it.
2. Provider can be changed at runtime. This will require the map to find the needed key in an object. But developer might know that there's only one map in an application or the provider won't be changed for some other reason. In this case, the developer can specify the key as a string